WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
```

Comment:
* When I see "alert", I think JavaScript alerts. Would wca_alert be a better name?

* You're using two different ways to do optional arguments. Why not use keyword arguments for both content and note? Maybe there is even a default value for type? So perhaps:

`def alert(type: :info, content: nil, note: false, &block)`

* This would benefit from a test!

* Some usage documentation could also be useful.